_data/sites.json

```diff
@@ -5388,6 +5388,17 @@
             "support.docusign.com"
         ]
     },
+    {
+        "name": "Doksure",
+        "url": "https://doksure.com/supprimer-son-compte",
+        "notes": "GDPR compliant. Go to Settings > Account > Delete Account.",
+        "notes_fr": "Conforme RGPD. Allez dans Paramètres > Compte > Supprimer son compte."
+        "difficulty": "easy",
+        "domains": [
+          "doksure.com"
+        ],
+        "email": "hello@doksure.com",
```

Comment:
```suggestion
```

We don't put the `e-mail` if it is not necessary for the deletion, as it might confuse users